actors/test/test_actor.go

```diff
@@ -0,0 +1,48 @@
+package test
```

Comment:
I think we can do better than `test` for the name, it'll be confusing with other things. I chatted briefly with @whyrusleeping and others and `puppet` came out as short and to the point.